### PR TITLE
feat: Enable event tracking

### LINKED ISSUE #43: Setup event tracking
We can just integrate segment during a pre-release, but before a developer preview we need to remove any 3rd party scripts, and proxy events instead (similar to how the Browser works).

There is already a function setup which gets called during all the events we want, it just isn't integrated with Segment yet.

### PR DESCRIPTION
**Pull request by kyranjamie**
Closes #43
Currently relies on an environment variable.

### CODE REVIEW

**Comment by hstove** [comment]
Hm - why does the git diff include everything in `refactor/screen-template`? It makes it much harder to review. Is there something off with the diff here?

**Comment by kyranjamie** [comment]
Fixed now — I had to `reset --hard refactor/screen-template` and cherry pick the commit back.

**Review by aulneau — APPROVED** [review]

File: webpack.config.js [hunk 1]
```diff
@@ -93,6 +94,8 @@ module.exports = {
       filename: 'actions.html',
       title: 'Blockstack',
       chunks: ['actions'],
+      segmentHost: 'cdn.segment.io',
```
Comment (hstove) [inline webpack.config.js]: Instead of using them here, let's just add this to `DefinePlugin`. That way, we won't have to repeat this for each HTML target.

**Review by hstove — CHANGES_REQUESTED** [review]
Very minor changes - then I'm happy to merge.

**Comment by hstove** [comment]
Note - I updated this PR to use `DefinePlugin` - feel free to push back if you think it should be different!

**Review by hstove — APPROVED** [review]

**Comment by kyranjamie** [comment]
No problem, co-author, looks better 

image

**Pull request merged**